documentation/src/main/asciidoc/topics/proc_configuring_hotrod_encryption.adoc

```diff
@@ -1,19 +1,42 @@
 [id='configuring-hotrod-encryption_{context}']
-= Configuring Hot Rod Client Encryption
+= Configuring Hot Rod client encryption
 {brandname} Server can enforce SSL/TLS encryption and present Hot Rod clients with certificates to establish trust and negotiate secure connections.
 
-To verify server-issued certificates, Hot Rod clients require part of the TLS
-certificate chain.
+To verify certificates issued to {brandname} Server, Hot Rod clients require part of the TLS certificate chain.
 For example, the following image shows a certificate authority (CA), named "CA", that has issued a certificate for a server instance named "HotRodServer":
 
 image::cert_chain.png[align="center", title="Certificate chain"]
 
+.Prerequisites
+
+* Create a trust store that lets Hot Rod clients verify {brandname} Server identities.
++
+The trust store is a Java keystore that contains part of the {brandname} Server certificate chain, typically the public certificate for the CA.
```

Comment:
We can lose "Java".

I think the client truststore must contain the public certificate of the server, not just the CA cert.

@tristantarrant Do we support server verification just via the CA in the client truststore?

Reply:
removed "Java"